public/config/configuration.json

```diff
@@ -4,20 +4,15 @@
   "AUTH_WEB_URL": "https://dev.bcregistry.ca/business/auth/",
   "AUTH_TOKEN_URL": "https://dev.oidc.gov.bc.ca/auth/realms/fcf0kpqr/protocol/openid-connect/token",
   "BUSINESSES_URL": "https://dev.bcregistry.ca/business/auth/",
-  "DEV_EXISTING_NR_TIMEOUT_MS": "0",
-  "DEV_NR_COMPLETION_TIMEOUT_MS": "0",
+  "DASHBOARD_URL": "https://dev.bcregistry.ca/business/",
```

Comment:
This is in `web-url` vault, `business` application, which is already being loaded (see ~/devops/vaults.json).